.github/workflows/Godot CI.yml

```diff
@@ -0,0 +1,34 @@
+name: Build Island.zip Artifact
+
+on: [push, pull_request]
+       
+env: 
+  GODOT_VERSION: 3.4.4 
+  EXPORT_NAME: Mod # directory where the Godot project is
+
+jobs:
+
+  export-linux:
+    name: Linux Export
+    runs-on: ubuntu-latest
+    container:
+      image: barichello/godot-ci:mono-latest # not sure this image works with godot 3.4.4
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          lfs: true
+      - name: Setup
+        run: |
+          mkdir -v -p ~/.local/share/godot/templates
+          mv /root/.local/share/godot/templates/${GODOT_VERSION}.stable ~/.local/share/godot/templates/${GODOT_VERSION}.stable
+      - name: Resolve Dependencies
+        run: |
+          mkdir -v -p build/linux
+          cd $EXPORT_NAME
+          timeout 30s godot . --editor --no-window 
```

Comment:
if this works then you do not have to move the templates at all FYI as they are unused when just zipping

Reply:
Thanks, I'll remove that step.